rpgame/src/main/java/pk/rpgame/view/FightView.java

```diff
@@ -26,23 +27,62 @@ private int getAction() {
   }
 
   public void printEnemies(List<Monster> monsters) {
-    System.out.println("Czeka cię walka z następującymi przeciwnikami: ");
+    System.out.println("These are the monsters that guard this room: ");
     for (Monster monster : monsters) {
       int i = 1;
-      System.out.println(i + ". " + monster.getName() + ", " + monster.getHealth() + " pkt życia");
+      System.out.println(i + ". " + monster.getName() + ", " + monster.getHealth() + " health points");
       i++;
     }
   }
 
   public void printHeroHealth(Hero hero) {
-    System.out.println("Posiadasz obecnie " + hero.getHealth() + " pkt życia");
+    System.out.println("You currently have " + hero.getHealth() + " health points");
+  }
+
+  public int getMonsterChoice(List<Monster> monsters) {
+    printEnemies(monsters);
+    System.out.print("Choose number of the monster you wish to attack: ");
+    int action = getAction();
+    while (true) {
+      if (action > monsters.size() || action < 1) {
+        System.out.println("Pick a correct number!");
+        action = getAction();
+      } else {
+        break;
+      }
+    }
+    return action;
+  }
+
+  public int getItemChoice(List<Item> items) {
+    System.out.println("Here are the items in your inventory: ");
+    for (Item item : items) {
+      int i = 1;
+      System.out.println(i + ". " + item.getName());
+      i++;
+    }
+    System.out.print("Choose number of the item you wish yo use: ");
+    int action = getAction();
+    while (true) {
+      if (action > items.size() || action < 1) {
+        System.out.println("Pick a correct number!");
+        action = getAction();
+      } else {
+        break;
+      }
+    }
+    return action;
+  }
```

Comment:
Jak coś to itemy podzieliłam na używalne (UsableItem), więc można przyjmować listę tylko takich itemów w view (i w kontrolerze je jakoś wyciągać osobno do osobnej listy), albo zrobić to tutaj -> sprawdzać czy Item jest klasy UsableItem. Choć chyba najlepiej w kontrolerze

Reply:
zmieniłem na przyjmowanie UsableItem, sprawdzanie lepiej w kontrolerze, widok najlepiej, żeby był tak głupi jak się da xd

Reply:
Nie widzę przeciwskazań